docs/8-veba-troubleshooting.md

```diff
@@ -1,20 +1,30 @@
 # vCenter Event Broker Appliance - Troubleshooting
 
+## Table of Contents
+[Requirements](#requirements)<br>
```

Comment:
Instead of these `<br>` statements to force newline either use two spaces after the text (this list) or just add a blank line in between the sections (see below).

Reply:
Added two spaces in the TOC.

Reply:
Ok, I'm not sure I fully understood the use of the <br> clauses. Was line-wrapping not working for you or did you just want more space between the lines?